Exiled.Events/Patches/Generic/IndividualFriendlyFire.cs

```diff
@@ -291,10 +279,7 @@ private static IEnumerable<CodeInstruction> Transpiler(IEnumerable<CodeInstructi
                 // targetReferenceHub
                 new(OpCodes.Ldloc_3),
 
-                new(OpCodes.Ldarg_0),
```

Comment:
This patch is disabled - why not just delete at this point? If it's needed again it's in the commit history

Reply:
Yea, you right

Reply:
Although no, it is marked as TODO, so let it exist(maybe ask dev bois later)

Reply:
> Although no, it is marked as TODO, so let it exist(maybe ask dev bois about removing it later)

That was Joker, and it was left there by me incase I screwed up the ff stuff for grenades, thus far I have not heard a report so, I am going to assume it's fine to remove. Though you can double check

Reply:
Sure, you right